geopyspark/geotrellis/rdd.py

```diff
@@ -401,6 +401,25 @@ def reproject(self, target_crs, extent=None, layout=None, scheme=FLOAT, tile_siz
 
         return TiledRasterRDD(self.geopysc, self.rdd_type, srdd)
 
+    def lookup(self, col, row):
+        """Return the value(s) in the image of a particular SpatialKey (given by col and row)
+
+        Args:
+            col: The SpatialKey column
+            row: The SpatialKey row
```

Comment:
It'd be nice if the docstrings were formatted similarly to the other docstrings in the file.

For instance, the `Args` would be formatted like:
```python
   col (int): The SpatialKey column
```